.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+
+sudo: false
+
+python:
+    - "2.7"
+
+virtualenv:
+    system_site_packages: true
+
+env:
+    matrix:
+        - LINT_CHECK="1"
+
+install:
+    - git clone --single-branch --depth=1 https://github.com/vauxoo/maintainer-quality-tools.git -b master ${HOME}/maintainer-quality-tools
+    - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+    - travis_install_nightly
+
+script:
+    - travis_run_tests
```

Comment:
@moylop260 
Thanks
I made another PR to fix this comment https://github.com/Vauxoo/SynRunbotWeblate/pull/6